apps/api/v2/src/modules/teams/memberships/controllers/teams-memberships.controller.ts

```diff
@@ -3,6 +3,7 @@ import { Roles } from "@/modules/auth/decorators/roles/roles.decorator";
 import { ApiAuthGuard } from "@/modules/auth/guards/api-auth/api-auth.guard";
 import { RolesGuard } from "@/modules/auth/guards/roles/roles.guard";
 import { OrganizationsEventTypesService } from "@/modules/organizations/services/event-types/organizations-event-types.service";
```

Comment:
we can remove this, not used anywhere

Reply:
Fixed in [refactor: logger name and unused import](https://github.com/calcom/cal.com/pull/18085/commits/6bf1ac90874af61b54b51469c34549b477804634)